src/server/config/schema.js

```diff
@@ -61,7 +61,7 @@ module.exports = Joi.object({
 
     events: Joi.any().default({}),
     dest: Joi.string().default('stdout'),
-
+    filter: Joi.any().default({}),
```

Comment:
Can we move [the defaults](https://github.com/elastic/kibana/pull/5038/files#diff-f998ed6969c535b593e16ebe899aab73R50) here please? With the current implementation you can't log authorization headers even if you wanted to.